x/blob/client/cli/tx.go

```diff
@@ -21,7 +21,7 @@ func GetTxCmd() *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	cmd.AddCommand(CmdWirePayForBlob())
-
+	cmd.AddCommand(CmdPayForBlob())
+	cmd.AddCommand(CmdTestRandBlob())
 	return cmd
```

Comment:
very unimportant optional formatting 
```suggestion
	cmd.AddCommand(CmdPayForBlob())
	cmd.AddCommand(CmdTestRandBlob())
	
	return cmd
```

Reply:
Reverted as gofumpt is angry if we don't leave new line for return statement 😅